manifests/install.pp

```diff
@@ -29,11 +29,22 @@
     }
   }
 
-  if $::foreman::passenger and $::foreman::passenger_ruby_package {
-    package { $::foreman::passenger_ruby_package:
-      ensure  => installed,
-      require => Class['apache'],
-      before  => Class['apache::service'],
+  if $::foreman::apache {
+    if $::foreman::passenger {
+      if $::foreman::passenger_ruby_package {
+        package { $::foreman::passenger_ruby_package:
+          ensure  => installed,
+          require => Class['apache'],
+          before  => Class['apache::service'],
+        }
+      }
+    } else {
+      # TODO: foreman-puma subpackage?
+      if $::osfamily == 'RedHat' {
```

Comment:
should this also gain maj==7 → tfm, else just rubygem, while we're at it?

Reply:
I should look closer at my own PRs. We now have a `foreman-service` package which should pull in all the dependencies. Will update.